docs/syntax-reference.rst

```diff
@@ -1006,11 +1006,10 @@ Scalar functions
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
 | FLOOR                  |  ``FLOOR(col1)``                                           | The floor of a value.                             |
 +------------------------+------------------------------------------------------------+---------------------------------------------------+
-| GEO_DISTANCE           | ``GEODISTANCE(lat1, lon1,                                  | Distance between two lat-lon points,              |
-|                        |               lat2, lon2,                                  | each specified in decimal degrees.                |
-|                        |               unit)``                                      | An optional final parameter can                   |
-|                        |                                                            | be used to specify either                         |
-|                        |                                                            | 'KM'(default) or 'Miles'.                         |
+| GEO_DISTANCE           |  ``GEO_DISTANCE(lat1, lon1, lat2, lon2, uint)``            | The great-circle distance between two lat-long    |
+|                        |                                                            | points, both specified in decimal degrees. An     |
+|                        |                                                            | optional final parameter specifies ``KM``         |
+|                        |                                                            | (the default) or ``MI``.                          |
```

Comment:
why put "MI" here ? the code will accept any of mi/mile/miles in order to be as helpful as it can but perhaps the one we list out in the docs should be the most intuitive one we can show ? reasonable people may disagree as to which one that is of course ;)  - but i would argue it's likely "miles", there's no international word or commonly-used abbreviation that i know of like "MI" - that only makes me think of millimeters!

Reply:
Sounds good, will change to "miles".

Reply:
Fixed in https://github.com/confluentinc/ksql/pull/1656.